verde/utils.py

```diff
@@ -207,6 +209,116 @@ def maxabs(*args, nan=True):
     return npmax(absolute)
 
 
+def build_grid(
+    coordinates,
+    data,
+    data_name,
+    dims=("northing", "easting"),
+    extra_coords_name=None,
+    attrs=None,
```

Comment:
Maybe remove this for now and we can be smarter about adding metadata to the Dataset later on. It's always easier to add later than to remove.